service_management/azure/test/integration/service_bus/queues_scenario_test.rb

```diff
@@ -13,18 +13,36 @@
 # limitations under the License.
 #--------------------------------------------------------------------------
 require 'integration/test_helper'
+require 'integration/service_bus/scenario_helper'
```

Comment:
I don't think so because it didn't find ScenarioHelper on line 44, so i added the helper file.